docs/05-module-block-5/02-learning-goals.adoc

```diff
@@ -14,54 +14,34 @@ Zudem vermittelt er Strategien, um mehrere Modelle, und die daraus erwachsenden
 
 [[LZ-5-2]]
 ==== LZ 5-2: Vor- und Nachteile eines Team-übergreifenden Modells abwägen können
-* Die TN kennen Conway's Law footnote:7[Cf.: <<conway>>] und können es auf die (Kommunikations-)Struktur von Entwicklungs-Teams und Software-Artefakten anwenden.
+* Die TN kennen Conway's Law footnote:7[Cf.: <<conway>>] und seine Auswirkungen auf Software-Architektur.
 * Die TN verstehen, dass von mehreren Teams genutzte Modelle teamübergreifende Kommunikation und Koordination (z. B. beim Deployment) erfordern.
 * Die TN verstehen, dass Modelle aufgeteilt werden müssen, damit mehrere Entwicklungsteams unabhängig voneinander arbeiten können.
 * Die TN verstehen, dass definierte Modellgrenzen eine wichtige Rolle dabei spielen, den Kommunikationsaufwand gering zu halten.
 * Die TN wissen, dass Begriffe der Fachsprache in verschiedenen Kontexten verschiedene Bedeutungen haben können.
 * Die TN wissen, dass lokale Modellkonsistenz unabhängig von anderen Modellen aufrechterhalten werden kann.
 
 [[LZ-5-3]]
-==== LZ 5-3: Modellgrenzen von Bounded Contexts in einer Context Map beschreiben können
+==== LZ 5-3: Vom Problem- zum Lösungsraum kommen können
+* Die TN kennen die Begriffe "Problem-" und "Lösungsraum" und den Unterschied.
+* Die TN verstehen im Lösungsraum "Modell" und "Design" und wie sie entstehen.
+* Die TN verstehen, wie im Problemraum die Domäne verstanden und in Subdomänen aufgeteilt werden kann.
```

Comment:
Eine allgemeine Anmerkung: Wie ihr sicher festgestellt habt, steht ca. 100 Mal 'Die TN verstehen/können/kennen' im Lehrplan. Wenn ihr wollt, könnt ihr das auch umformulieren und einfach weglassen. Beispiel hier: 

- Begriffe "Problemraum" und "Lösungsraum" kennen und voneinander unterscheiden können
- Die Begriffe "Modell" und "Design" im Lösungsraum verstehen und deren Entstehung kennen.
- Die Domäne im Problemraum verstehen und in Subdomänen aufteilen können.

Reply:
Ich denke, dass ist etwas für zukünftige Aktualisierungen.

Reply:
Ich denke, dass ist etwas für zukünftige Überarbeitungen.